example/angular_components_example/build.yaml

```diff
@@ -21,3 +21,4 @@ targets:
           - --minify
           - --trust-type-annotations
           - --trust-primitives
+          - --dump-info
```

Comment:
Also add this

```yaml
      build_web_compilers|dart2js_archive_extractor:
        options:
          filter_outputs: false
```

So the dump info file actually goes through...

Reply:
Done.